src/components/users/ResetPasswordForm.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Loader from 'react-loader-spinner';
+import { object, string } from 'prop-types';
+
+import { userActions } from '../../actions/user.actions';
+import '../../style/App.scss';
+import './user-form.scss';
+import FormInput from '../common/FormInput';
+import { userRequest } from '../../actions/user.actions';
+
+const ResetPasswordForm = ({ intl, url_uid, url_token }) => {
```

Comment:
since you went for camel case at the beginning of the project I would keep that convention. You can decamelize object later when needed. [humps](https://github.com/domchristie/humps)